client/platform/desktop/backend/native/viame.ts

```diff
@@ -212,6 +213,75 @@ async function runPipeline(
   return jobBase;
 }
 
+/**
+ * a node.js implementation of dive_tasks.tasks.export_trained_model
+ */
+async function exportTrainedPipeline(settings: Settings,
+  exportTrainedPipelineArgs: ExportTrainedPipeline,
+  updater: DesktopJobUpdater,
+  validateViamePath: (settings: Settings) => Promise<true | string>,
+  viameConstants: ViameConstants,
+): Promise<DesktopJob> {
+  const { path, pipeline } = exportTrainedPipelineArgs;
+
+  const isValid = await validateViamePath(settings);
+  if (isValid !== true) {
+    throw new Error(isValid);
+  }
+
+  const exportPipelinePath = npath.join(settings.viamePath, PipelineRelativeDir, "convert_to_onnx.pipe");
```

Comment:
While in most instances VIAME is installed along with DIVE it isn't always and the versions between VIAME and DIVE might not align.  This is to say that there isn't a guarantee that the `convert_to_onnx.pipe` file exists.  There should be a clear error to the user if this is the case.